manual/de/ch_Tools_Assistants.xml

```diff
@@ -5278,15 +5278,15 @@
                   <glossentry>
                     <glossterm>Ask (Briefkurs)</glossterm>
                     <glossdef>
-                      <para>Gibt an, zu welchem Preis der Händler bereit sind, eine Aktie zu verkaufen.
+                      <para>Gibt an, zu welchem Kurs der Händler bereit ist, eine Aktie zu verkaufen.
                       </para>
                     </glossdef>
                   </glossentry>
 
                   <glossentry>
                     <glossterm>Last (Zuletzt)</glossterm>
                     <glossdef>
-                      <para>Gibt den Kurs an, zu dem der letzte Handel mit dieser Aktie/Wertpapier vollzogen wurde. Dieser Preis
+                      <para>Bezeichnet den Kurs, zu dem der letzte Handel mit dieser Aktie/Wertpapier vollzogen wurde. Dieser Preis
                         wird meistens in den Medien genannt.
```

Comment:
`Dieser Preis wird meistens in den Medien genannt.`
Verwechselst du den nicht mit close? 
Gut, zwischen Börsenschluß Eröffnung am nächstem Handelstag sind die gleich, aber whrend der Öffnungszeiten gibt es `last`, aber kein  `close`.

Vielleicht fügst du noch eine Liste ein, von Daten die FQ zwar zur Verfügung stellt, aber von GC entweder gemapped oder ignoriert werden (volume).

Einige werden gern in einem Symbol zusammengefaßt:
Open/Close: Balken links, rechts
High/Low als vertikaler Balken

Reply:
> Vielleicht fügst du noch eine Liste ein, von Daten die FQ zwar zur Verfügung stellt, aber von GC entweder gemapped oder ignoriert werden (volume).

Ich habe die fehlenden Kursarten zusätzlich aufgeführt. Ich spiele noch mit dem Gedanken, die Beschreibungen der Informationen die F::Q abruft, auch dorthin zu verschieben

Reply:
Richtig!
Für den Anfang kannst du auch auf die Labels in 'man Finance::Quote' verweisen.